viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -0,0 +1,457 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import TWEEN from '@tweenjs/tween.js';
+import { ComboControls } from './ComboControls';
+import { CallbackData } from './types';
+
+export class CameraManager extends THREE.EventDispatcher {
```

Comment:
Why does this inherit EventDispatcher?

Reply:
I thought that it could be similar to how ComboControls throw 'cameraChange' event and because of that inherit EventDispatcher class. Probably can remove it for now

Reply:
Yeah, remove it - not used currently.